apps/frontpage/components/docs/mdx/home-renderers.tsx

```diff
@@ -21,10 +21,17 @@ export function HomeRenderers() {
         subtitle="with Webpack"
         title="React"
       />
+      <Card
+        href="/docs/get-started/frameworks/react-native-web-vite/?renderer=react-native-web"
+        logo="logo-react.svg"
+        subtitle="in browser"
```

Comment:
Something like this would be more consistent with the other cards. Wdyt?

```suggestion
        subtitle="with Vite (in browser)"
```